src/parser/parser.yy

```diff
@@ -1349,7 +1349,39 @@ yield_columns
     ;
 
 yield_column
-    : expression {
+    : KW_VERTEX {
+        $$ = nullptr;
+        throw nebula::GraphParser::syntax_error(@1, "please add alias when using vertex.");
+    }
+    | KW_VERTEX KW_AS name_label {
+        $$ = new YieldColumn(VertexExpression::make(qctx->objPool()), *$3);
+        delete $3;
+    }
+    | KW_VERTICES {
+        $$ = nullptr;
+        throw nebula::GraphParser::syntax_error(@1, "please add alias when using vertices.");
+    }
```

Comment:
If the following rules are formulated, can't this situation be automatically disabled?

Reply:
in order to give users some tips